src/lib/variants.ts

```diff
@@ -251,6 +254,42 @@ export const hasNoTicker: Filter = {
     },
 };
 
+// Edge case filter to exclude any test where the combination of targeting data and variants logic
+// would allow the test to pass, but the value used to replace the article count
+// in the Epic copy is "0" (zero) due to all article views having been
+// within the`periodInWeeks` defined in the test (used for targeting)
+// but before (i.e. older) than the hardcoded `52` weeks (used for template rendering).
+// Prevents an Epic from ever rendering with text "...you've read 0 articles...".
+// This is needed because we use different timeframes when finding a
+// test/variant and when replacing the Epic placeholder, so we should
+// have a specific check around the logic/values used for template rendering.
+// This is intentional and in parity with Frontend.
+export const hasNoZeroArticleCount = (now: Date = new Date()): Filter => ({
+    id: 'hasNoZeroArticleCount',
+    test: (test, targeting): boolean => {
+        // Check if the test requires an article history
+        const mustHaveHistory =
```

Comment:
Does the comment add anything here? The variable name is descriptive enough on its own I'd argue.

Reply:
Cool - updated